python/lsst/obs/base/makeRawVisitInfo.py

```diff
@@ -161,15 +200,23 @@ def offsetDate(self, date, offsetSec):
         return DateTime(dateNSec + int(offsetSec*1.0e9), DateTime.TAI)
 
     def popItem(self, md, key, default=None):
-        """Remove an item of metadata and return the value
-
-        @param[in,out] md  metadata, as an lsst.daf.base.PropertyList or PropertySet;
-            the popped key is removed
-        @param[in] key  metadata key
-        @param[in] default  default value to return if key not found; ignored if doRaise true
-        @return the value of the specified key, using whatever type md.get(key) returns
-
-        Log a warning if the key is not found
+        """Remove an item of metadata and return the value.
+
+        Log a warning if the key is not found.
+
+        Parameters
+        ----------
+        md : `lsst.daf.base.PropertyList` or `PropertySet`
+            Metadata to pull `key` from and remove.
+        key : `str`
+            Metadata key to extract.
+        default : `object`
+            Default value to return if key not found; ignored if doRaise true
```

Comment:
Please fix an existing error: there is no `doRaise` so this should read "Value to return if key not found" (the word Default seems redundant)